bftengine/src/bftengine/RequestsBatchingLogic.cpp

```diff
@@ -28,7 +28,9 @@ RequestsBatchingLogic::RequestsBatchingLogic(InternalReplicaApi &replica,
       maxInitialBatchSize_(config.maxInitialBatchSize),
       batchFlushPeriodMs_(config.batchFlushPeriod),
       maxNumOfRequestsInBatch_(config.maxNumOfRequestsInBatch),
+      initialMaxNumOfRequestsInBatch_(config.maxNumOfRequestsInBatch),
       maxBatchSizeInBytes_(config.maxBatchSizeInBytes),
+      adaptiveConsensusSize_(config.adaptiveConsensus),
```

Comment:
adaptiveConsensus is boolean; why do you call it adaptiveConsensusSize_?

Reply:
Only God knows why, changed thanks.